src/workerd/jsg/value-test.c++

```diff
@@ -1304,5 +1307,45 @@ KJ_TEST("jsg::Name works") {
   e.expectEval("forSymbolShared('foo') !== forSymbolApi('foo')", "boolean", "true");
 }
 
+// ========================================================================================
+
+struct RustContext: public ContextGlobalObject {
+  kj::String getRustString(::rust::String value) {
```

Comment:
Like the other types of string values, should `::rust::String` support `jsg::NonCoercible<::rust::String>`? Not critical if there's not an immediate use case.